x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/start_step.tsx

```diff
@@ -242,7 +299,7 @@ export const StartStep: React.FC<StartStepProps> = ({
                   }
                 }}
                 fill
-                disabled={!canConfigureConnector}
+                disabled={!canConfigureConnector || !isValidIndexName(rawName)}
```

Comment:
In all of these checks, we're not taking into account the `error` prop being propagated into this component. We probably sould.

Reply:
I also added `error` as an argument to evaluate when we do these checks